Makefile.toml

```diff
@@ -877,9 +877,9 @@ dependencies = ["prepare"]
 condition = { env_set = [
   "ENABLE_BUILD_RW_CONNECTOR",
 ], files_modified = { input = [
-  "./java/connector-node/**/*.java",
+  "./java/connector-node/**/*",
```

Comment:
Updates on `pom.xml` and `resources` should also trigger recompiling.

Reply:
How about the `target` folder that contains the build output? Is it possible for cargo make to ignore changes in files in .gitignore?

Reply:
Do you mean including the `target` dir or excluding it? 👀